File: mcp_servers/js/clients/src/client_and_server_config.ts
export const ClientsConfig:any = [
    "MCP_CLIENT_OPENAI",
    "MCP_CLIENT_AZURE_AI",
    "MCP_CLIENT_GEMINI",
    // "CLAUDE",
]

export const ServersConfig:any = [
    {
        server_name :"WORDPRESS",
        server_features_and_capability:`WORDPRESS`,
        path : "build/index.js"
    },
    {
        server_name :"GITHUB",
        server_features_and_capability:`GitHub server provides comprehensive repository management, file operations, issue tracking, pull request management, search functionality, and more. Supports creating/updating files, managing branches, creating issues/PRs, searching code/users/repositories, and handling Git operations.`,
        path : "dist/index.js"
    },
    {
        server_name :"ATLASSIAN",
        server_features_and_capability:`Atlassian Trello server provides comprehensive project management capabilities including board management, list operations, card management, team collaboration, and workflow automation. Supports creating/updating cards, managing lists, board operations, activity tracking, file attachments, and workspace management.`,
        path : "dist/index.js"
    },
    {
        server_name :"TABLEAU",
        server_features_and_capability:`Tableau MCP server provides comprehensive data visualization and analytics capabilities. Supports listing published data sources, executing VizQL queries, accessing metadata, and managing Tableau Server/Cloud resources. Enables AI assistants to interact with Tableau dashboards and data.`,
        path : "dist/index.js"
    },
]

File: mcp_servers/python/clients/src/client_and_server_config.py
ClientsConfig =[
    "MCP_CLIENT_AZURE_AI",
    "MCP_CLIENT_OPENAI",
	"MCP_CLIENT_GEMINI"
]
ServersConfig = [
	# {
	# 	"server_name": "MCP-GSUITE",
	# 	"command":"uv",
	# 	"args": [
	# 		"--directory",
	# 		"../servers/MCP-GSUITE/mcp-gsuite",
	# 		"run",
	# 		"mcp-gsuite"
	# 	]
	# },
    {
		"server_name": "AWS-EC2",
		"server_features_and_capability": "AWS EC2 MCP server provides comprehensive AWS CLI integration with tools for executing AWS commands, getting help documentation, and managing AWS resources. Supports command pipelines with Unix utilities, AWS resource management, security auditing, cost optimization, and infrastructure automation following AWS Well-Architected Framework best practices.",
		"command": "uv",
		"args": [
			"--directory",
			"../servers/AWS-EC2",
			"run",
			"python",
			"-m",
			"aws_mcp_server"
		]
	},
	{
		"server_name": "TEAMSPEAK",
		"server_features_and_capability": "TeamSpeak MCP server provides comprehensive TeamSpeak server management capabilities including user management, channel operations, server administration, messaging, and real-time monitoring. Supports creating/deleting channels, managing user permissions, sending messages, kicking/banning users, and retrieving server statistics and client information.",
		"command": "python",
		"args": [
			"-m",
			"teamspeak_mcp.server"
		],
		"cwd": "../servers/TEAMSPEAK"
	},
    {
		"server_name": "WHATSAPP",
        "server_features_and_capability": "WhatsApp MCP server provides comprehensive WhatsApp Business API integration including messaging, group management, session handling, and chat history. Supports sending text messages, creating/managing groups, adding/removing participants, retrieving chat lists and message history, and managing WhatsApp API sessions through GreenAPI.",
		"command": "uv",
		"args": [
			"--directory",
			"../servers/WHATSAPP",
			"run",
			"whatsapp-mcp"
		]
	}
]
